lessons/session_3/session_3_training.md

```diff
@@ -0,0 +1,35 @@
+---
+marp: true
+theme: uncover
+_class: invert
+paginate: true
+output: pdf_document
+---
+
+# ML Systems in Production
+# Session 3: Training
+Jesús A. Calderón
+February 2024
+
+---
+
+# Agenda
+
+**1.1 Overview of ML Systems**
+	
++ Sampling
```

Comment:
Would this section have any similarities to our Sampling course that we are offering?

Reply:
I think it does. This is a small section on how to use sampling instead of working with the entire data set when, eventually, the intention is to work with the entire thing. Will leave it in, since it is part of the book and I do not expect that it will be a huge overlap. One difference with the other course that I saw is that this implementation is in Python, where I belive the other one is in R.